src/components/templates/NewsAndEvents.html

```diff
@@ -0,0 +1,106 @@
+<div class="news-events__wrap container">
```

Comment:
![image](https://user-images.githubusercontent.com/83479175/206902207-60ef1a2a-5d3f-4f59-a7e7-d1a6fdb5add6.png)

`container`の外に背景が適用されていないので、背景用の要素でcontainerを囲む必要があります。

```suggestion
<div class="news-events__wrap">
　<div class="container">
```

Reply:
こちら確認お願いします 🙏